os/board/rtl8721csm/src/Makefile

```diff
@@ -141,6 +141,7 @@ CFALGS += -I$(DIR_CRYPTO)/../include/
 CFLAGS += -I$(DIR_FW)/../include/
 CFLAGS += -I$(TOPDIR)/net/netmgr/
 CFLAGS += -I$(TOPDIR)/include/net/if
+CFLAGS += -I$(TOPDIR)/include/tinyara/netmgr	
```

Comment:
I worry about exceeding the limitation of CFLAGS and our policy is using like below instead of adding this CFLAGS.
```
#include <tinyara/netmgr/aaa.h>
```

Reply:
Same as line 143.
```
#include <net/if/bbb.h>
```

Reply:
Thanks sincerely, Mr Chang. I have commited the changes. May I check with you that this policy is applicable to headers under folder (TOPDIR)/include/ ?

Reply:
There are three basic paths for header.
1. os/include
2. framework/include
3. external/include
You can include the headers which are located in apps/include like below
```
i.e to include https://github.com/Samsung/TizenRT/blob/master/apps/include/system/sysinfo.h
#include <apps/system/sysinfo.h>
```
If some headers are under three basic path, you had better include with full path.
I mean
i.e
header location : os/include/aaa/bbb/ccc.h
then #include <aaa/bbb/ccc.h>

If something is not under them, then you can add CFLAGS

Reply:
Actually I don't want to make a limitation to use CFLAGS for board specific folder. But as you know, CFLAGS has a length limitation and this folder already has too many CFLAGS. If you don't have any issue on it, you can do as you want.

Reply:
Got it, thank you a lot for your kind explanation!